packages/repository-modernizer/resources/pom.xml

```diff
@@ -23,7 +23,8 @@
     <packaging>pom</packaging>
     <version>${version}</version>
     <description>${appTitle}</description>
-
+    <parent>
```

Comment:
What happens if there is not parent pom info in pom, will the empty tags remain here, or will they be deleted?

Reply:
they will be deleted